modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptClientCodegen.java

```diff
@@ -792,6 +789,12 @@ public void processOpts() {
 
         additionalProperties.putIfAbsent(FILE_CONTENT_DATA_TYPE, propPlatform.equals("node") ? "Buffer" : "Blob");
 
+        if (!propPlatform.equals("deno")) {
```

Comment:
I don't know deno at all, but why did you decide not to generate `package.mustache` and `tsconfig.mustache` for deno? 
This makes the required dependencies intransparent.

Reply:
Thanks for your review !

Deno does not use `package.json` in its module resolution algorithm(Deno uses "ES Modules". Third party modules are imported via URLs.).
And also generated code can work by Deno's default tsconfig without optional `tsconfig.json`.